jitbuilder/lljb/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+###############################################################################
+# Copyright (c) 2019, 2019 IBM Corp. and others
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which accompanies this
+# distribution and is available at http://eclipse.org/legal/epl-2.0
+# or the Apache License, Version 2.0 which accompanies this distribution
+# and is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# This Source Code may also be made available under the following Secondary
+# Licenses when the conditions for such availability set forth in the
+# Eclipse Public License, v. 2.0 are satisfied: GNU General Public License,
+# version 2 with the GNU Classpath Exception [1] and GNU General Public
+# License, version 2 with the OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] http://openjdk.java.net/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+#############################################################################
+
+if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
+    set(ENV{PATH} "/usr/local/opt/llvm\@8/bin/:${PATH}")
+endif()
+
+find_package(LLVM 8 REQUIRED CONFIG)
+message(STATUS "Found LLVM ${LLVM_PACKAGE_VERSION}")
+message(STATUS "Using LLVMConfig.cmake in: ${LLVM_DIR}")
+#include_directories(${LLVM_INCLUDE_DIRS})
+
+add_library(lljb SHARED
+    src/Compiler.cpp
+    src/MethodBuilder.cpp
+    src/Module.cpp
+    src/IRVisitor.cpp
+)
+
+get_property(JITBUILDER_INCLUDE TARGET jitbuilder PROPERTY INTERFACE_INCLUDE_DIRECTORIES)
+
+target_include_directories(lljb
+    PUBLIC
+        include/
+        ${JITBUILDER_INCLUDE}
+        ${CMAKE_CURRENT_SOURCE_DIR}/../../compiler
+        ${CMAKE_CURRENT_SOURCE_DIR}/../..
```

Comment:
I would express this as `${omr_SOURCE_DIR}/compiler`. Isn't `../..` just the root of OMR? You probably don't need that.